website/docs/docs/deploy/model-notifications.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Model notifications"
+description: "Receive notifications about failures with your models. "
+---
+
+# Model notifications <Lifecycle status="beta" />
+
+Set up dbt to notify the appropriate owners &mdash; through Slack, email, or both &mdash; about any failures your models have encountered. When models fail, your reports are no longer accurate and the data can no longer be trusted so it's important to resolve these failures quickly.
```

Comment:
This is email only for the beta for the time being

Reply:
Can we add something here about the user value? My stab at it "Model level notifications mean that model owners can always find out about issues (whether the be warning tests, failing tests, or failing models) before your stakeholders do. Because these are more granular, you'll get notified about these as soon as they occur--even while the job is still running"

Reply:
You might need to tone done my sales tone to be docs appropriate :)

Reply:
:) folded this in. did i capture what you wanted to say appropriately?